koku/masu/util/gcp/common.py

```diff
@@ -84,7 +84,11 @@ def match_openshift_resources_and_labels(data_frame, cluster_topologies, matched
 
         if resource_id_df.any():
             LOG.info("Matching OpenShift on GCP by resource ID.")
-            ocp_matched = resource_id_df.str.contains("|".join(matchable_resources))
+            matching_check = "|".join(matchable_resources)
+            if matching_check:
+                ocp_matched = resource_id_df.str.contains(matching_check)
+            else:
+                continue
```

Comment:
nitpick: we could reverse the check here
```suggestion
            if not matching_check:
                continue
            ocp_matched = resource_id_df.str.contains(matching_check)
```

Reply:
My brain would not let me think of this and I tried really hard! LACK OF SLEEP SUCKS!! I know you would pick it up though! Thank you!! :heart: